templates/frontend/pages/search.tpl

```diff
@@ -18,76 +18,74 @@
  *}
 {include file="frontend/components/header.tpl" pageTitle="common.search"}
 
-<div class="container page-search">
-	<div class="page-header">
-		<h1>
-			{if $query}
-				{translate key="plugins.themes.healthSciences.search.resultsFor" query=$query|escape}
-			{elseif $authors}
-				{translate key="plugins.themes.healthSciences.search.resultsFor" query=$authors|escape}
-			{else}
-				{translate key="common.search"}
-			{/if}
-		</h1>
-	</div>
+<div class="container">
+	<h1>
+		{if $query}
+			{translate key="plugins.themes.healthSciences.search.resultsFor" query=$query|escape}
+		{elseif $authors}
+			{translate key="plugins.themes.healthSciences.search.resultsFor" query=$authors|escape}
+		{else}
+			{translate key="common.search"}
+		{/if}
+	</h1>
 	<div class="row justify-content-lg-center">
-		<div class="col-lg-8 search-col-results">
-			<div class="search-results">
+		<div class="col-lg-8">
+			<div>
 
 				{* No results found *}
 				{if $results->wasEmpty()}
 					{if $error}
-						<div class="alert alert-danger" role="alert">{$error|escape}</div>
+						<div role="alert">{$error|escape}</div>
```

Comment:
Class alert and alert-danger that is deleted here brings the styling from Bootstrap. Do you want to drop it? If yes, are considering to discriminate error message from others? The same question about alert-info class on line 40.

Reply:
Also will create a separate component for that!

Reply:
Created in separate issue - #16